functionary/inference.py

```diff
@@ -1,12 +1,19 @@
 from typing import Dict, List, Optional, Union
 
 import torch
+from lmformatenforcer import CharacterLevelParser, JsonSchemaParser
+from lmformatenforcer.integrations.vllm import build_vllm_logits_processor
```

Comment:
lmformatforcer only works for server_vllm, right ? because server.py also uses this file so I am not sure if we should put the code for handling the logits processor in this file or not